### PR TITLE
Update README.zh-CN.md

### PR DESCRIPTION
**Pull request by byronhe**

### CODE REVIEW

File: README.zh-CN.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ Dragonfly 支持 Redis 的常见参数。
 
 * `admin_bind`：如果设置，将管理控制台 TCP 连接绑定到给定地址。支持 HTTP 和 RESP 协议。默认为any。
 
-* `admin_nopass`: 将管理控制台 TCP 连接绑定到给定地址。同时支持 HTTP 和 RESP 协议。
+* `admin_nopass`: 如果设置，允许在不提供任何认证令牌的情况下，通过指定的端口访问管理控制台。同时支持 HTTP 和 RESP 协议。
```
Comment (Niennienzz) [inline README.zh-CN.md:122]: 建议在末尾增加默认值说明 `默认为 false。`

**Review by Niennienzz — COMMENTED** [review]
Thanks for contributing! Left a minor suggestion.

File: README.zh-CN.md (hunk 1 above)
Comment (byronhe) [reply README.zh-CN.md]: done

**Review by Niennienzz — APPROVED** [review]
LGTM

@romange needs approval by reviewers with write access 👍

**Pull request merged**